lib/css/components/_stacks-badges.less

```
@@ -105,3 +105,38 @@
 .s-badge__important {
     .badge-styles(transparent, var(--red-600), var(--white));
 }
+
+//  $$  Award Count
+//  ---------------------------------------------------------------------------
+.award-bling {
+    display: flex;
+    align-items: center;
+    color: inherit;
+
+    &:before {
+        content: "";
+        margin-top: -1px;
+        margin-right: 3px;
```

```suggestion
        margin-right: 4px;
```

Looking a little tight with the standard HTML spacing.